### DISCUSSION TITLE
Integrating with an existing ESLint setup

### DISCUSSION THREAD

**Discussion by Skn0tt** [ec0lint/ec0lint#68] [General]
Hey there! This looks like an amazing tool, and I'd love to introduce into one of my projects. I already have ESLint fully set up, and I wondered wether there's a way I could integrate `ec0lint` with that? e.g. is there something like an eslint/ec0lint plugin?

**Comment by malwinq** [comment]
Hi @Skn0tt ! I'm sorry we didn't reply earlier, your discussion must have got lost in other messages :(

Unfortunately there's no plugin of ec0lint to eslint, it's a separate project with its own root, as it comes with CO2 calculation module. However, the setup should be almost the same as eslint! Maybe worth trying? :) We're more than happy to get your feedback on installation + configuration process.
